test/mpi/coll/testlist.in

```diff
@@ -56,6 +56,7 @@ coll10 4
 coll11 4
 coll12 4
 coll13 4
+largebuf
```

Comment:
does the test need to be run with more than one process? if should, you should add the number after the test name

Reply:
That's not really necessary because overflows are not because of communication. I will force 2 processes, just in case.

Reply:
Ah, ok. In that case 1 process is probably preferred, especially for debugging, but you can just explicitly set it to 1?